src/util/settings.h

```diff
@@ -24,19 +26,31 @@ namespace util {
 //!       https://github.com/bitcoin/bitcoin/pull/15934/files#r337691812)
 using SettingsValue = UniValue;
 
-//! Stored bitcoin settings. This struct combines settings from the command line
-//! and a read-only configuration file.
+//! Stored bitcoin settings. This struct combines settings from the command line,
```

Comment:
style-nit: Should say PACKAGE_NAME, but that seems a bit too verbose. Might as well remove

```suggestion
//! Stored settings. This struct combines settings from the command line,
```

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15935#discussion_r458620301

> style-nit: Should say PACKAGE_NAME, but that seems a bit too verbose. Might as well remove

Thanks, updated